src/Symfony/Component/Lock/Store/FlockStore.php

```diff
@@ -108,7 +109,7 @@ private function lock(Key $key, $blocking)
      */
     public function putOffExpiration(Key $key, $ttl)
     {
-        // do nothing, the flock locks forever.
+        throw new NotExpirableStoreException();
```

Comment:
I wonder if it's should be treated like a BC break. ping @nicolas-grekas

Reply:
Looks like a BC break to me, but might be worth doing anyway instead of silently ignoring the call.